src/client/core/utils/get-time-limited-promise.js

```diff
@@ -1,8 +1,18 @@
 import { Promise } from '../deps/hammerhead';
 import delay from './delay';
-import { timeLimitedPromiseTimeoutExpired } from '../../../errors/runtime/message';
+import { RUNTIME_ERRORS } from '../../../errors/types';
+import TEMPLATES from '../../../errors/runtime/templates';
 
+const timeLimitedPromiseTimeoutExpiredTemplate = TEMPLATES[RUNTIME_ERRORS.timeLimitedPromiseTimeoutExpired];
+
+class TimeLimitedPromiseTimeoutExpiredError extends Error {
```

Comment:
Unfortunately, it's impossible. This will cause the webmake error `Native node.js modules are not ported to client-side.`.

Client `get-time-limit-promise.js` references `runtime/errors.js`, that references `create-stack-filter.js`, that references the standard node module `path`.

Reply:
Ok, I see